specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,987 @@
+                  "example": "multiadsample"
+              },
+              "storageEndpointSuffix": {
+                  "type": "string",
+                  "description": "",
+                  "example": "blob.core.windows.net"
+              },
+              "sourceFilePath": {
+                  "type": "string",
+                  "description": "",
+                  "example": "/data"
+              },
+              "dataSchema": {
+                  "description": "Model status. One of CREATED, RUNNING, READY, and FAILED.",
+                  "$ref": "#/definitions/ModelStatus"
+              },
+              "startTime": {
+                  "type": "string",
+                  "format": "date-time",
+                  "description": "A required field, indicating the start time of training data. Should be date-time.",
+                  "example": "2019-04-01T00:00:00Z"
+              },
+              "endTime": {
+                  "type": "string",
+                  "format": "date-time",
+                  "description": "A required field, indicating the end time of training data. Should be date-time.",
+                  "example": "2019-04-02T00:00:00Z"
+              },
+              "displayName": {
+                  "type": "string",
+                  "description": "An optional field. The name of the model whose maximum length is 24.",
+                  "example": "DevOps-MultiAD",
+                  "maxLength": 24
+              },
+              "slidingWindow": {
+                  "type": "integer",
+                  "description": "An optional field, indicating how many previous points will be used to compute the anomaly score of the subsequent point.",
+                  "example": 20
+              },
+              "alignPolicy": {
+                  "description": "An optional field, indicating the manner to align multiple variables.",
+                  "$ref": "#/definitions/AlignPolicy"
+              },
+              "status": {
+                  "description": "Model status. One of CREATED, RUNNING, READY, and FAILED.",
+                  "$ref": "#/definitions/ModelStatus"
+              },
+              "errors": {
+                  "type": "array",
+                  "description": "Error messages when failed to create a model.",
+                  "items": {
+                      "$ref": "#/definitions/ErrorResponse"
+                  }
+              },
+              "diagnosticsInfo": {
+                  "description": "Diagnostics Information for inspecting model/variable states.",
+                  "$ref": "#/definitions/DiagnosticsInfo"
+              }
+          },
+          "type": "object",
+          "description": "Training result of a model including its status, errors and diagnostics information."
+      },
+      "ModelStatus": {
```

Comment:
Do we need to add a `Deleted` status to let users know which models have been marked as deleted?

Reply:
No need.